sale_quotation_capable_vendor_rfq/models/sale_order.py

```diff
@@ -0,0 +1,165 @@
+# Copyright (C) 2024 Cetmix OÜ
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from collections import defaultdict
+
+from odoo import _, api, fields, models
+from odoo.exceptions import UserError
+
+
+class SaleOrder(models.Model):
+    _inherit = "sale.order"
+
+    rfq_ids = fields.One2many(
+        "purchase.order",
+        "sale_quotation_id",
+        string="RFQs",
+        help="RFQs created from this sale order",
+    )
+    rfq_count = fields.Integer(
+        string="RFQ count",
+        compute="_compute_rfq_count",
+        help="Number of RFQ created from this sale order",
+    )
+
+    @api.depends("rfq_ids")
+    def _compute_rfq_count(self):
+        """
+        Compute the number of RFQs created from this sale order
+        """
+        for order in self:
+            order.rfq_count = len(order.rfq_ids)
+
+    def action_view_rfq(self):
+        """
+        View the purchase orders created from this sale order
+        """
+        self.ensure_one()
+        action = self.env.ref("purchase.purchase_rfq").read()[0]
+        action["domain"] = [("id", "in", self.rfq_ids.ids)]
+        return action
+
+    def action_create_rfq(self):
+        """
+        Create purchase orders from this sale order
+        """
+        self.ensure_one()
+        self._create_rfq()
+        if self.rfq_count:
+            return self.action_view_rfq()
+        raise UserError(_("No RFQs created"))
+
+    def _get_capable_vendors(self, day_min=0, day_max=0):
```

Comment:
```suggestion
    def _get_capable_vendors(self, days_min=0, days_max=0):
```

Please ensure the names match the previous changes.